libs/community/langchain_community/embeddings/zhipuai.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any, Dict, List
+
+from langchain_core.embeddings import Embeddings
+from langchain_core.pydantic_v1 import BaseModel, Field, root_validator
+from langchain_core.utils import get_from_dict_or_env
+
+
+class ZhipuAIEmbeddings(BaseModel, Embeddings):
+    """ZhipuAI embedding models.
+
```

Comment:
Could you include a URL to the API so users can learn more about it from looking at the code?

Reply:
Done